### PR TITLE
docs: guide developers towards better efficiency

### PR DESCRIPTION
**Pull request by zolotokrylin**

### CODE REVIEW

File: .github/CONTRIBUTING.md
Comment (angelicawill) [inline .github/CONTRIBUTING.md:1]: There are some typos in line 36. 

Corrected:
> Once a Problem is identified, we report it as a [GitHub Issue](https://docs.github.com/en/issues) with the following naming pattern: `Problem: [statement]`. We’re counting on our contributors to identify problems. Keep a Problem name short (under 65 chars) and crystal clear.

This is out of scope of this PR so feel free to dismiss it if you prefer to keep it separate, and I'll open another PR to correct the typo if needed.

Comment (zolotokrylin) [reply .github/CONTRIBUTING.md:1]: @angelicawill did you make a commit? Or can you propose the change so I can easily apply it.

Comment (angelicawill) [reply .github/CONTRIBUTING.md:1]: @zolotokrylin You didn't touch those lines in this PR, hence I'm not able to add suggestion involving those lines. I don't have access to commit to your branch as well.

I'll open a PR to fix the typo

Comment (angelicawill) [reply .github/CONTRIBUTING.md:1]: Here's the PR: https://github.com/holdex/developers/pull/21 @zolotokrylin

**Review by Addis-Belete — APPROVED** [review]

**Comment by Addis-Belete** [comment]
The above-reported issue can be merged into the main as a separate PR.

**Pull request merged**